src/index.js

```diff
@@ -5,10 +5,15 @@ var EMPTY_OBJ = {}
 var EMPTY_ARR = []
 var map = EMPTY_ARR.map
 var isArray = Array.isArray
-var defer =
-  typeof requestAnimationFrame !== "undefined"
-    ? requestAnimationFrame
-    : setTimeout
+var defer =(() => {
```

Comment:
Using an arrow function here would drop support for some of the older browsers.

Reply:
> 在此处使用箭头功能会放弃对某些旧版浏览器的支持。

Yeap, it should be function(), but this PR seems can not merged